spec/factories/cohorts.rb

```diff
@@ -10,13 +10,16 @@
     registration_start_date { Date.new(start_year.to_i, 5, 10) }
     academic_year_start_date { Date.new(start_year.to_i, 9, 1) }
 
+    trait :previous do
+      start_year { Date.current.year - (Date.current.month < 9 ? 2 : 1) }
```

Comment:
Im not too sure about the rules for cohort here, similar to the question above and if we should avoid using September?

Reply:
Given that I explicitly set Sept, 1st as the academic_year_start_date in the factory, I am using that same date to compute the previous cohort in the factory. 
Maybe I could rewrite it like this:
```
trait :previous do
      start_year { Date.current.year - (Date.current.month < academic_year_start_date.month ? 2 : 1) }
end
```

Reply:
Controversial thought, but should the cohort be in code (ie a config file) rather than in the database?

Clearly not one for now.